MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/maps/MapboxMapOptionsAttrsTest.kt

```diff
@@ -75,6 +75,30 @@ class MapboxMapOptionsAttrsTest {
     }
   }
 
+  @Test
+  fun getFontFamilyNoMode() {
+    mockEnableLocalIdeograph(enabled = true)
+    val font = "foo"
+    mockLocalIdeographString(font)
+    every { typedArray.getInt(R.styleable.mapbox_MapView_mapbox_glyphRasterizationMode, 0) } returns 0
+    val options = MapboxMapOptions.createFromAttributes(options, context, typedArray)
+    verify(exactly = 1) {
+      options.setRasterizationMode(GlyphsRasterizationMode.NO_GLYPHS_RASTERIZED_LOCALLY)
+    }
+  }
+
+  @Test
+  fun getFontFamilyIdeographGlyphMode() {
+    mockEnableLocalIdeograph(enabled = true)
+    val font = "foo"
+    mockLocalIdeographString(font)
+    every { typedArray.getInt(R.styleable.mapbox_MapView_mapbox_glyphRasterizationMode, 0) } returns 1
+    val options = MapboxMapOptions.createFromAttributes(options, context, typedArray)
+    verify(exactly = 1) {
+      options.setRasterizationMode(GlyphsRasterizationMode.IDEOGRAPHS_RASTERIZED_LOCALLY)
+    }
+  }
+
```

Comment:
Similar test for `IDEOGRAPHS_RASTERIZED_LOCALLY` mode?

Reply:
I guess is `ALL_GLYPHS_RASTERIZED_LOCALLY`, right?